lib/WebService/SendBird/GroupChat.pm

```diff
@@ -170,4 +170,24 @@ sub update {
     return $self
 }
 
+=head2 set_freeze
+
+Freeze or unfreeze the channel.
+
+See L<https://sendbird.com/docs/chat/v3/platform-api/guides/group-channel#2-freeze-a-channel>.
+
+=cut
+
+sub set_freeze {
+    my ($self, $freeze) = @_;
+    
+    $freeze = $freeze ? \1 : \0;
```

Comment:
just curious, why the `\` in front of 1 and 0?

Reply:
They are shortcuts for `JSON::PP::true` and `JSON::PP::false`: https://metacpan.org/pod/JSON::PP#JSON::PP::true,-JSON::PP::false

Reply:
Cool, thanks